django/chat/management/commands/eval_responses.py

```diff
@@ -255,9 +255,9 @@ def _create_test_user():
 def _test_qa_response(eval_instance, user):
     try:
         # Create chat and build chat history
-        chat = Chat(user=user)
+        chat = Chat.objects.create(user = user, mode = eval_instance["mode"])
         chat.save()
-        chat_options = ChatOptions.objects.create(chat=chat, mode=eval_instance["mode"])
+        chat_options = ChatOptions.objects.get(chat=chat)
```

Comment:
With the above change, the custom create method creates the ChatOptions object by default, so you only have to get it here.